models/articles.models.js

```diff
@@ -10,3 +10,18 @@ exports.fetchArticleId = (article_id) => {
       return res.rows[0];
     });
 };
+
+exports.fetchAllArticles = (sort_by = "created_at") => {
+  let query = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url,  COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id`;
+
+  const allowedQuery = ["created_at"];
```

Comment:
You don't need any of this logic to handle the query but I think keep it in, as it's only allowing for a _created_at_ anyway, and it might come in handy later in the week...
but be careful to not give yourself extra work! If the ticket doesn't ask for something, don't include it